R/zzz.R

```diff
@@ -14,7 +14,9 @@
                  "`install.packages('fivethirtyeightdata', ",
                  "repos = 'https://fivethirtyeightdata.github.io/drat/', type = 'source')`")
     msg <- paste(strwrap(msg), collapse="\n")
-    packageStartupMessage(msg)
+    packageStartupMessage(msg) }
+  else {
+    library(fivethirtyeightdata)
```

Comment:
@rudeboybert if the user has `fivethirtyeightdata` already installed, this code will make it autoload when calling `library(fivethirtyeight)` (`library(fivethirtyeightdata)` will not show in the console though). 
I'm not sure how good of a solution this is with regards to software engineering principles -- but it is something that works.
The official way to do this is by indicating the package as a depends in the DESCRIPTION, but since 538data won't be on CRAN, we can't do this.

Reply:
I think this is fine. I think [`USAboundaries`](https://github.com/ropensci/USAboundaries/blob/953c144a60445eabe9b8a0c4ac7720725f1fc23b/R/install-data-pkg.R) does this. As long as we are prompting people at the installation step of `538data` we are good to go